cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/MainnetParams.hs

```diff
@@ -0,0 +1,70 @@
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Cardano.Testnet.Test.MainnetParams
+  ( hprop_mainnet_params
+  ) where
+
+import           Cardano.Testnet
+
+import qualified Data.Aeson as A
+import qualified Data.Aeson.Lens as A
+import qualified Data.ByteString.Lazy.Char8 as B
+import           Data.Default.Class (def)
+import           Lens.Micro ((^?))
+import qualified System.Process as IO
+
+import           Testnet.Process.Run (execCli', mkExecConfig)
+import           Testnet.Property.Util (integrationRetryWorkspace)
+import           Testnet.Start.Types (CreateEnvOptions (..), GenesisOptions (..),
+                   UserProvidedEnv (..), TestnetOnChainParams (..))
+
+import           Hedgehog ((/==))
+import qualified Hedgehog as H
+import qualified Hedgehog.Extras as H
+
+-- | Execute me with:
+-- @DISABLE_RETRIES=1 cabal test cardano-testnet-test --test-options '-p "/Can get on-chain parameters from mainnet/"'@
+hprop_mainnet_params :: H.Property
+hprop_mainnet_params = integrationRetryWorkspace 2 "mainnet-params" $ \tmpDir -> H.runWithDefaultWatchdog_ $ do
+
+  let testnetOptions = def { cardanoOutputDir = UserProvidedEnv tmpDir }
+      genesisOptions = def { genesisEpochLength = 200 }
+      createEnvOptions = def {ceoOnChainParams = OnChainParamsMainnet}
```

Comment:
```suggestion
      createEnvOptions = def {ceoOnChainParams = OnChainParamsMainnet}
```
Unfortunately we should not do http requests in tests, because it makes them dependent on the environment, which makes them non-deterministic. That's why it's failing in hydra now.